src/SDKs/_metadata/iotcentral_resource-manage.txt

```diff
@@ -0,0 +1,35 @@
+﻿Installing AutoRest version: latest
+
```

Comment:
Please delete this file and regenerate code

Reply:
I've cleaned this up